src/cli.js

```diff
@@ -681,6 +702,54 @@ var exports = {
   },
 
   /**
+   * Helper to get proper reporter.
+   *
+   * @param {string} name of reporter to fetch, pre-defined or custom.
+   */
+  getReporter: function(reporterName) {
+    var reporter = defReporter; // default
+
+    if (!reporterName) {
+      return reporter; // return default
+    }
+    switch (true) {
```

Comment:
Want me to switch it to switch(reporterName)? You're right the original switch was (true) because it had multiple variable comparison.